actix/src/sec-5-arbiter.md

```diff
@@ -15,89 +15,148 @@ for your actors to run inside. When you call `start()` on your actor it is then
 running inside of the System Arbiter's thread. In many cases, this is all you
 will need for a program using Actix.
 
+While it only uses one thread, it uses the very efficient event loop pattern
+which works well for asynchronous events. To handle synchronous tasks that
+require heavy CPU power, it's better to avoid blocking the event loop and
```

Comment:
I don't really like the phrasing "heavy cpu power" or "synchronous tasks". I think better phrasing could be "to process tasks that are long lived and/or concurrent, it can be better to have other threads process these tasks instead. For this situation ..."

Reply:
I agree and will correct.

Reply:
Thank you! Really appreciate you wanting to help with the documentation here!

Reply:
CPU-bound is right term for CPU heavy execution

Reply:
It's more the phrasing here is complex/unnatural rather than exclusion of the term "CPU". "heavy cpu power" is odd to me.

Reply:
I think the new phrasing addresses all concern.